src/fromager/commands/build.py

```diff
@@ -19,7 +19,7 @@
     wheels,
 )
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('')
```

Comment:
Let's not rename this one, and just get the root logger in the function where we need to use it.

Reply:
Okay, will do